.github/workflows/test.yml

```diff
@@ -0,0 +1,85 @@
+name: Build and Test
+
+on:
+  merge_group:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/master' }}
+
+env:
+  NODE_VERSION: 18
```

Comment:
Is it possible to keep testing this on 14 and 16? https://github.com/auth0/node-jwks-rsa/blob/master/.circleci/config.yml#L40